extensions/amp-lightbox-gallery/1.0/component.jss.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createUseStyles} from 'react-jss';
+
+const gallery = {
+  position: 'absolute !important',
+  left: '0 !important',
+  right: '0 !important',
+  top: '0 !important' /* Matches height of top-bar */,
+  height: '100%',
+  width: '100%',
+  bottom: '0 !important',
+  overflow: 'auto !important',
+};
+
+const controlsPanel = {
+  position: 'absolute !important',
+  height: '56px !important' /* Matches top of gallery */,
+  width: '100% !important',
+  zIndex: '1',
+  background: 'linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0))',
+  '@media (min-width:1024px)': {
+    height: '80px !important' /* Matches top of gallery */,
+  },
+};
+
+const lightbox = {
+  '&$showControls $control': {
+    animationTimingFunction: 'ease-in',
+    animationName: '$fadeIn',
+  },
+  '&$hideControls $control': {
+    animationTimingFunction: 'linear',
+    animationName: '$fadeOut',
+  },
+};
+
+const showControls = {};
+const hideControls = {};
+
+const control = {
+  animationFillMode: 'forwards',
+  animationDuration: '400ms',
+  position: 'absolute !important',
+  boxSizing: 'content-box',
+  cursor: 'pointer !important',
+  zIndex: '2',
+};
+
+const topControl = {
+  width: '24px',
+  height: '24px',
+  padding: '16px',
+  '@media (min-width:1024px)': {
+    width: '40px',
+    height: '40px',
+    padding: '20px',
+  },
```

Comment:
nit: these are all pixels by default when numbers
```suggestion
  width: 24,
  height: 24,
  padding: 16,
  '@media (min-width:1024px)': {
    width: 40,
    height: 40,
    padding: 20,
  },
```

Consider also sharing a constant for `20` or `40` across properties.

Reply:
Thanks, done!